content/topics/dotenv-module/_index.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Introduction to Dotenv module 
+ready: true
+
+---
+
+## What is an environment variable?
+
+An [environment variable](https://en.wikipedia.org/wiki/Environment_variable) is a variable whose value is set outside the program, typically through functionality built into the operating system or microservice. An environment variable is made up of a name/value pair, and any number may be created and available for reference at a point in time.
+
+```
+//example 
+NAME_OF_PLACE=Umuzi
+
+```
+
+## How does dotenv help you?
+
+Using environment variables is one technique to make your app easier to configure by separating infrequently changing data from your code. [Dotenv](https://www.npmjs.com/package/dotenv) module loads environment variables from a .env file that you create and adds them to the [process.env](https://nodejs.org/docs/latest/api/process.html#process_process_env) object that is made available to the application
+
```

Comment:
It would be nice to add a little bit more info about the importance of separating code from configuration.